### DISCUSSION TITLE
When would you move the main branch to lit 2

### DISCUSSION THREAD

**Discussion by emexal-company** [adobe/spectrum-web-components#1846] [Q&A] [answered]
Lit 2 stable release is available since the 22 Sep. What are your plans to move to lit 2 ? Is there anyway to use lit 2 based components, using NPM (without need to build manually the components) ?

**Comment by Westbrook** [comment, marked as answer]
We are tracking the move to `lit@2.0` here: https://github.com/adobe/spectrum-web-components/pull/1401 I can promise when we’ll be making the conversion in our `latest` release channel, but it will be soon. In the interim, we maintain a `lit-next` release channel on NPM that features the code in that PR. 

Could you share some more information as to what you mean by “manually build the components”?

**Reply by emexal-company** [reply]
"manually build the components” => build the lit-next branch my-self, and use packages localy, not from NPM. I wasn't aware of the lit-next channel in NPM.

**Reply by Westbrook** [reply]
Awesome. Please let us know how they work out for you. Hopefully sometime in November they will be come `latest` and then everyone can have them!

**Reply by Westbrook** [reply]
In case you missed it, as of November 8th we moved the `lit@2.0` branch to `main` and are releasing all of our versions going forward based on the new versions of those libraries.
